packages/assets-controllers/src/NftController.test.ts

```diff
@@ -149,6 +184,24 @@ function setupController({
     getNetworkClientById,
   );
 
+  const getInternalAccountMock = jest
```

Comment:
This should be renamed to `mockGetInternalAccount` or `mockGetAccount`, cause right now our mocks follow 2 different naming pattern `mock*` and `*Mock` which makes it kind of weird to read/follow.